book/course/conda.md

```diff
@@ -0,0 +1,540 @@
+# Conda package manager
+
+Content from this lesson has been inspired and adapted from a number of sources including:
+- [Conda documentation](https://docs.conda.io/en/latest/)
+- [Carpentries Incubator: Introduction to conda for data scientists](https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/)
```

Comment:
```suggestion
- [Carpentries Incubator: Introduction to Conda for data scientists](https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/)
```

I'm not chasing all these down, but just check you're capitalising Conda everytime it's not the command.